### ISSUE TITLE
QueryParams should support wait time in minutes, not just seconds.

### ISSUE THREAD

**Issue by duncanscott** [Ecwid/consul-api#123]
QueryParams should support wait time in minutes, not just seconds.  According to the consul documentation on blocking queries:

[https://www.consul.io/api/index.html#blocking-queries](https://www.consul.io/api/index.html#blocking-queries)

> In addition to index, endpoints that support blocking will also honor a wait parameter specifying a maximum duration for the blocking request. This is limited to 10 minutes. If not set, the wait time defaults to 5 minutes. This value can be specified in the form of "10s" or "5m" (i.e., 10 seconds or 5 minutes, respectively).

**Comment by vgv** [comment]
Hi,

I hope I understand you correctly, but why you can't use seconds to express minutes? 

For example, if you want to wait for 3 minutes (for example), just pass 180 to QueryParam constructor. 180 seconds is 3 minutes.

**Comment by duncanscott** [comment]
Yes, I tried that.  Any minute value can be expressed in seconds.

I thought that the timeout error on a blocking query might be due to consul
accepting only minutes for longer blocking periods.  It seems that the
timeout error had another source.

Thank you for this valuable library.

**Comment by vgv** [comment]
I think that timeouts have been fixed here - #125
This fix is available in the latest release 1.2.4

**Comment by baozi-2019** [comment]
I used version 1.4.5
```java
catalogClient.getCatalogService(serviceName, CatalogServiceRequest.newBuilder().setQueryParams(QueryParams.Builder.builder().setWaitTime(10).build()).build());
```
This is my code. I want him to connect for a maximum of ten second, then disconnect or throw an exception after timeout. Now it seems that setting a timeout is not useful.
